internal/cluster/sbd.go

```diff
@@ -86,23 +100,37 @@ func NewSBD(executor utils.CommandExecutor, cluster, sbdPath, sbdConfigPath stri
 	return s, nil
 }
 
-func getSBDConfig(sbdConfigPath string) (map[string]interface{}, error) {
-	sbdConfFile, err := os.Open(sbdConfigPath)
+func LoadSbdConfig(sbdConfigPath string) (map[string]string, error) {
+	sbdConfigFile, err := os.Open(sbdConfigPath)
 	if err != nil {
-		return nil, errors.Wrap(err, "could not open sbd config file")
+		return nil, &SBDConfigLoadingError{
```

Comment:
Do we need such a tailored error handling?
Why don't we return how we were returning before?
`errors.Wrap(err, "could not open sbd config file")`
It looks like an overkill to me